ourjapanlife/findadoc-web
# Use Pinia State for Modals

Resolves #212 

# What changed
1. @pinia/nuxt was added to better facilitate using Pinia to manage the state for components.
2. modalStore.ts was created to manage the Modal.vue component's state
3.  `.env` was added to .gitignore.

# Testing instructions
1. Rename app.vue to app-backup.vue and  app-copy.vue to app.vue in order to see the main page.
4. Change `isOpen: false` to `isOpen: true` in _modalStore.ts_ so the modal opens by default when you refresh the page.
![Screenshot 2023-07-17 at 7 17 04 PM](https://github.com/ourjapanlife/findadoc-web/assets/31802656/e90ce3a0-4cb4-433c-98f5-fcbbc9326961)

5. Start the dev server and confirm that clicking the X on the modal closes it on all Chrome, Firefox, and Safari.
